src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -532,11 +533,11 @@ protected void sendPutResults(UUIDBase uuid, long context, long version,
                                     results.size(), 
                                     mgBase.getMyID(), storageState, deadlineRelativeMillis); // FUTURE - allow constructor without this?
             if (debug) {
-                System.out.println("results.size "+ results.size());
+                Log.fineAsyncf("results.size: %d"+ results.size());
             }
             for (PutResult result : results) {
                 if (debug) {
-                    System.out.println(result);
+                    Log.fineAsync(result);
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect